index.html

```diff
@@ -1982,13 +1983,11 @@ <h3>
         <p>
           The steps for <dfn>processing the <code>start_url</code> member</dfn>
           are given by the following algorithm. The algorithm takes a
-          <a>USVString</a> <var>value</var>, a <a>URL</a> <var>manifest
-          URL</var>, and a <a>URL</a> <var>document URL</var>. This algorithm
-          returns a <a>URL</a>.
+          <a>USVString</a> <var>value</var>, and a <a>URL</a> <var>manifest
```

Comment:
```suggestion
          {{USVString}} <var>value</var>, and a <a>URL</a> <var>manifest
```

Reply:
I didn't want to do this here; everything else in the surrounding text is in the "old style" so I'm leaving it (for now) for consistency. I think this could all be modernized in a separate PR.